pysite/pysite/settings.py

```diff
@@ -29,11 +29,11 @@
 SECRET_KEY = config["Secret_Key"]
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+#DEBUG = False
```

Comment:
Gotta be a way to set this up so you don't have to manually change it every time. NB precommit hooks.